ui/locales/en/common.json

```diff
@@ -44,7 +44,7 @@
   "welcomeTitle": "Welcome",
   "welcomeDesc": "Welcome to the future of space exploration with MoonDAO! Be part of our community-driven initiative to democratize access to space and accelerate our multiplanetary future. Mission Control is your gateway to all things MoonDAO.",
   "mooneyTitle": "Get $MOONEY",
-  "mooneyDesc": "Getting $MOONEY is now easier than ever with a simple login, wallet creation, and onboarding flow that will walk you through everything you need to join our borderless, permissionless community.",
+  "mooneyDesc": "Getting $MOONEY is now easier than ever with a simple swap.",
```

Comment:
Can you talk to Ryan about adding something more in-depth here.

The subtitle should:
- Have a pointer to the docs to explain MOONEY
- Have a pointer to the - "funding" flow so they can get ETH (and get a wallet if they don't have one)
- Have a pointer to the "next step" with getting them to lock for voting power.